qa/TL3_YOLO_convergence/test_tensorflow.sh

```diff
@@ -16,14 +16,29 @@ export NCCL_NVLS_ENABLE=0
 # workaround for https://github.com/tensorflow/tensorflow/issues/63548
 export WRAPT_DISABLE_EXTENSIONS=1
 
+export DATA_DIR=/data/coco/coco-2017/coco2017/
```

Comment:
Nitpick: adding trailing `/` to directory names leads to some unexpected behaviors from time to time. It should work here, though.

Reply:
Fixed